app-kotlin/build.gradle

```diff
@@ -65,10 +69,10 @@ dependencies {
   implementation 'com.squareup.moshi:moshi-kotlin:1.13.0'
 
 //  Kotlin Coroutines
-  implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.3'
-  testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.3'
+  implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4'
+  testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.4'
 
-//  Picasso
+//  picasso
```

Comment:
is this intended?

Reply:
not intended is there any way to avoid such single space change in commit ?